From 3603ea38cdff71a84aa7a8d7bd08b153a7d29552 Mon Sep 17 00:00:00 2001
From: leigh123linux <leigh123linux@fedoraproject.org>
Date: Tue, 30 Sep 2014 12:35:26 +0100
Subject: [PATCH] rename

---
 Makefile-insttest.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile-insttest.am b/Makefile-insttest.am
index efc2a71..b08ff10 100644
--- a/Makefile-insttest.am
+++ b/Makefile-insttest.am
@@ -3,7 +3,7 @@ EXTRA_DIST += \
 	installed-tests/script.test.in \
 	$(NULL)
 
-installedtestmetadir = $(datadir)/installed-tests/gjs
+installedtestmetadir = $(datadir)/installed-tests/cjs
 installedtestmeta_DATA = 
 if BUILDOPT_INSTALL_TESTS
 installedtestmeta_DATA += jsunit.test
-- 
1.9.3

