From c7cb693a4e91b05f8ccd8237307f16e1923f7be6 Mon Sep 17 00:00:00 2001
From: leigh123linux <leigh123linux@fedoraproject.org>
Date: Fri, 2 Jun 2017 11:08:24 +0100
Subject: [PATCH] tweener: Add undefined property check (#45)

---
 modules/tweener/tweener.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules/tweener/tweener.js b/modules/tweener/tweener.js
index 782cb268..e648c9e6 100644
--- a/modules/tweener/tweener.js
+++ b/modules/tweener/tweener.js
@@ -542,7 +542,7 @@ function _addTweenOrCaller(target, tweeningParameters, isCaller) {
                 copyProperties[istr] = new PropertyInfo(properties[istr].valueStart,
                                                         properties[istr].valueComplete,
                                                         properties[istr].valueComplete,
-                                                        properties[istr].arrayIndex,
+                                                        properties[istr].arrayIndex || 0,
                                                         {},
                                                         properties[istr].isSpecialProperty,
                                                         properties[istr].modifierFunction,
