--- a/Makefile.am
+++ b/Makefile.am
@@ -39,6 +39,7 @@ include gjs-srcs.mk
 nobase_gjs_public_include_HEADERS = $(gjs_public_headers)
 
 ########################################################################
+pkgconfigdir = $(libdir)/pkgconfig
 pkgconfig_DATA = cjs-1.0.pc
 
 EXTRA_DIST += cjs-1.0.pc.in			\
--- a/configure.ac
+++ b/configure.ac
@@ -28,7 +28,6 @@ AM_SILENT_RULES([yes])
 
 # our first pkg-config invocation is conditional, ensure macros still work
 PKG_PROG_PKG_CONFIG
-PKG_INSTALLDIR
 
 AC_LANG([C++])
 AC_PROG_CXX
@@ -109,7 +108,9 @@ AS_IF([test x$have_gtk = xyes], [
 ], [AS_IF([test "x$with_gtk" = "xyes"],
   [AC_MSG_ERROR([GTK requested but not found])])])
 
-PKG_CHECK_VAR([GI_DATADIR], [gobject-introspection-1.0], [gidatadir])
+ 
+GI_DATADIR=$($PKG_CONFIG --variable=gidatadir gobject-introspection-1.0)
+AC_SUBST(GI_DATADIR)
 
 AC_SUBST([CJS_PACKAGE_REQUIRES])
 AC_SUBST([CJS_PACKAGE_REQUIRES_PRIVATE])
